packages/commands/src/components/command-menu.js

```diff
@@ -53,9 +54,14 @@ function CommandMenuLoader( { name, search, hook, setLoader, close } ) {
 					>
 						<HStack
 							alignment="left"
-							className="commands-command-menu__item"
+							className={ classnames(
+								'commands-command-menu__item',
+								{
+									'with-no-icon': ! command.icon,
```

Comment:
Core uses the `has-*` naming convention for similar helper classes. Maybe we should follow it here as well.

Reply:
Yea, good point I'll reverse the logic.